Content.Client/Stylesheets/IStylesheetManager.cs

```diff
@@ -1,12 +1,14 @@
+using Content.Client.Stylesheets.Redux;
 using Robust.Client.UserInterface;
 
-namespace Content.Client.Stylesheets
+namespace Content.Client.Stylesheets;
+
+public interface IStylesheetManager
 {
-    public interface IStylesheetManager
-    {
-        Stylesheet SheetNano { get; }
-        Stylesheet SheetSpace { get; }
+    Stylesheet SheetNanotransen { get; }
+    Stylesheet SheetSystem { get; }
```

Comment:
I've thought about this for a bit and I think it would also be worthwhile to keep `StyleNano.cs` and `StyleSpace.cs` as they were, but just obselete them. Makes merging easier.